app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -55,68 +55,6 @@ class Meta:
         )
 
 
-class NimbusChangeLogMixin:
```

Comment:
I kind of liked these functional bits split out into mixins. But I guess if they're not doing much, it saves some boilerplate & overhead to merge them in and just keep the more complex pieces split out 🤷‍♂️

Reply:
Yeah no I feel you, in this case it was because when we had separate serializers, the changelog logic was shared between all of them but now it's only used in the single one and it's only two lines.  Technically none of them need to be mixed in but the heavier ones like the branches and probe sets it's nice to have the logic pulled out so it's easier to read, but this one, so smol.